docs/sphinx_doc/en/source/tutorial/206-prompt.md

```diff
@@ -300,6 +300,55 @@ print(prompt)
 ]
 ```
 
+
+### LiteLLMChatWrapper
+
+`LiteLLMChatWrapper` encapsulates the litellm chat API, which takes a list of
+messages as input. The litellm support different types of models, and each model
```

Comment:
support - > supports? And please fix the typo "as weel as" below

Reply:
Thanks for pointing it out. Fixed.